tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -53,9 +53,8 @@ int InternalCommandsHandler::execute(uint16_t clientId,
 }
 
 void InternalCommandsHandler::addMetadataKeyValue(SetOfKeyValuePairs &updates, uint64_t sequenceNum) const {
-  concord::kvbc::BlockMetadata metadata(*m_storage);
-  Sliver metadataKey = metadata.getKey();
-  Sliver metadataValue = metadata.serialize(sequenceNum);
+  Sliver metadataKey = m_blockMetadata->getKey();
```

Comment:
Just a note that this doesn't match the google style guide which uses suffix underscores rather than `m_` prefixes. However, this does conform to the existing style of the file so I support this choice. We should change all of the member variables together in another commit.

Reply:
Thanks Andrew - indeed, I preferred to conform to the existing code in this instance. We can clean this up in a separate effort.